README.Rmd

```diff
@@ -156,7 +164,9 @@ Alternatively, you can install the development version from GitHub.
 remotes::install_github("r-prof/proffer")
 ```
 
-To use functions `pprof()` and `serve_pprof()`, you need to install [`pprof`](https://github.com/google/pprof). Installing `pprof` is hard, so if you have trouble, please do not hesitate to [open an issue](https://github.com/r-prof/proffer/issues) and ask for help. And if you cannot install `pprof`, then [`profvis`](https://rstudio.github.io/profvis/) is an excellent alternative.
+To use functions `pprof()` and `serve_pprof()`, [`pprof`](https://github.com/google/pprof) needs to be installed. Installing `pprof` is hard, so if you have trouble, please do not hesitate to [open an issue](https://github.com/r-prof/proffer/issues) and ask for help. And if you cannot install `pprof`, then [`profvis`](https://rstudio.github.io/profvis/) is an excellent alternative.
+
+As you follow the installation instructions below, you can run `test_pprof()`, `assert_pprof()`, or `pprof_path()` at any time to see if `proffer` can find and use `pprof`. If these functions succeed early, you are already done.
 
 1. Install the [`RProtoBuf`](https://github.com/eddelbuettel/rprotobuf) package. On Linux, you also need to install the supporting protocol buffer libraries, e.g. `sudo apt-get install protobuf-compiler libprotobuf-dev libprotoc-dev` on Ubuntu.
```

Comment:
{RProtoBuf} is a strong dependency now. Any problems that have occurred here will already have occurred when installing the package.

```suggestion
```